### PR TITLE
document responsible disclosure process

### PR DESCRIPTION
**Pull request by JonasCir**
Include a `SECURITY.md` which states how we handle responsible disclosure as discussed internally.
Kudos to @chThie for creating the file!
@markusmann-vg @MartinWahnschaffeSymeda @kwa20

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Our team and community take all security bugs in SORMAS seriously.
+Thank you for improving the security of SORMAS. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.  
+Unfortunately, SORMAS does not offer a paid bug bounty programme or other forms of compensation.
+
+Report security bugs by emailing at **security@sormas.org**.
+
+We will acknowledge your email within 48 hours, and will send a
```
Comment (MartinWahnschaffeSymeda) [inline SECURITY.md:22]: We will acknowledge your email and follow up with a response within 10 business days, or explain why a reply may take longer. The response will indicate the next steps in handling your report.

**Review by MartinWahnschaffeSymeda — CHANGES_REQUESTED** [review]
@JonasCir Just one change request

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+Our team and community take all security bugs in SORMAS seriously.
+Thank you for improving the security of SORMAS. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.  
+Unfortunately, SORMAS does not offer a paid bug bounty programme or other forms of compensation.
+
+Report security bugs by emailing at **security@sormas.org**.
+
+We will acknowledge your email and follow up with a response within 10 business days, or explain why a reply may take longer. The response will indicate the next steps in handling your report.
```
Comment (MartinWahnschaffeSymeda) [inline SECURITY.md:22]: Sorry. I meant to keep the second part of this. So please re-add:

After the initial reply to your report, the security team will endeavor to keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.

**Review by MartinWahnschaffeSymeda — CHANGES_REQUESTED** [review]
@JonasCir @chThie

File: SECURITY.md (hunk 2 above)
Comment (chThie) [reply SECURITY.md:22]: Ok, sorry. I though the second sentence was a replacement for it. I will re-add it now.

**Comment by MartinWahnschaffeSymeda** [comment]
Merged. Thanks a lot @chThie and @JonasCir

**Pull request merged**